lib/share_progress/button.rb

```diff
@@ -21,7 +35,8 @@ def destroy(id)
 
       def all(options={})
```

Comment:
I did not realize until just now you could do this in ruby. It's something I've wished they'd had for a while. :+1: from me.